main.tf

```diff
@@ -91,7 +91,7 @@ module "lambda" {
   handler                        = "${local.lambda_handler}.lambda_handler"
   source_path                    = var.lambda_source_path != null ? "${path.root}/${var.lambda_source_path}" : "${path.module}/functions/notify_slack.py"
   recreate_missing_package       = var.recreate_missing_package
-  runtime                        = "python3.11"
+  runtime                        = "python3.12"
```

Comment:
we should not add both a python version bump and support for a new notification in same PR

Reply:
done ! reverted it back to python 3.11.